cognite/extractorutils/rest/extractor.py

```diff
@@ -106,6 +123,8 @@ def __init__(
         cancelation_token: threading.Event = threading.Event(),
         config_class: Type[CustomRestConfig] = RestConfig,
         use_default_state_store: bool = True,
+        config_file_path: Optional[str] = None,
+        num_parallel_requests: int = 10,
```

Comment:
could be useful to put into configurable properties?

Reply:
Yeah, I guess it makes more sense as config option.